subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -914,13 +956,68 @@ u8_t ll_adv_enable(u8_t enable)
 				   &ll_adv_ticker_ext[handle]
 #endif /* CONFIG_BT_TICKER_EXT */
 				   );
+
+#if (CONFIG_BT_CTLR_ADV_AUX_SET > 0)
+		if (lll->aux) {
+			struct lll_adv_aux *lll_aux = lll->aux;
+
+			ret = ull_ticker_status_take(ret, &ret_cb);
+			if (ret != TICKER_STATUS_SUCCESS) {
+				goto failure_cleanup;
+			}
+
+			aux = (void *)HDR_LLL2EVT(lll_aux);
+			ull_hdr_init(&aux->ull);
+			aux->interval =
+				adv->interval +
+				(HAL_TICKER_TICKS_TO_US(ULL_ADV_RANDOM_DELAY) /
+				 625U);
```

Comment:
yes, its intentional, as primary PDU drifts by an amount of random delay. By having a fix value greater than the max random delay, the total auxiliary PDU count is less than or equal to total primary PDUs. This way every auxiliary PDU gets referenced by atleast one primary PDU.

Reply:
But won't there then be also primary PDUs that point to an auxiliary PDU that will not be sent because it conflicts with another primary PDU?

Reply:
auxiliary PDU being of larger interval than its primary PDU, gets precedence in the collision resolution. The skipped primary PDU gets precedence in its next interval; in case of repeated overlap, primary and auxiliary PDUs round-robin in schedule ensuring the tuples are scheduled in air.